crypto_api/CryptoApiManager.py

```diff
@@ -1,18 +1,17 @@
 from datetime import datetime, timedelta
-
 import aiohttp
 import grequests
-
 import config
 from services.deposit import DepositService
 
 
 class CryptoApiManager:
-    def __init__(self, btc_address, ltc_address, trx_address, eth_address, user_id):
+    def __init__(self, btc_address, ltc_address, trx_address, eth_address, sol_address, user_id):
         self.btc_address = btc_address.strip()
         self.ltc_address = ltc_address.strip()
         self.trx_address = trx_address.strip()
         self.eth_address = eth_address.strip().lower()
+        self.sol_address = sol_address.strip()
```

Comment:
maybe separate this addition to a different branch if you want to introduce new crypto tokens?

Reply:
@6R1M5L07H I've already put this feature in a separate branch.
https://github.com/ilyarolf/AiogramShopBot/pull/64